articles/azure-arc/servers/ssh-arc-overview.md

```diff
@@ -20,11 +20,25 @@ SSH access to Arc-enabled servers provides the following key benefits:
 
 ## Prerequisites
 
+- User Permissions: Owner or Contributor role assigned for the target Arc-enabled server.
```

Comment:
This is not necessarily required. You need Owner or contributor to set the default endpoint and service configuration, but once those are set up you don't need that much permission. If I'm not mistaken you need "Virtual Machine Local User Login"

Reply:
I see that is what was documented before, so I guess it means that's the requirement for setting up the feature. But it's a little confusing.